src/house/houses/houseList.js

```diff
@@ -27,7 +27,7 @@ const HouseList = () => {
   } else generalHouses = totalHouses;
   useEffect(() => {
     dispatch(findHousesThunk());
-    setTimeout(() => dispatch(findPublicHousesThunk()), 500);
+    //setTimeout(() => dispatch(findPublicHousesThunk()), 500);
```

Comment:
nit: we might need it line uncommented in main, thanks

Reply:
addressed